frontend/src/Episode/Summary/MediaInfo.tsx

```diff
@@ -16,9 +17,11 @@ function MediaInfo(props: MediaInfoProps) {
           return null;
         }
 
-        return (
-          <DescriptionListItem key={key} title={title} data={props[key]} />
-        );
+        if (key === 'audioBitrate' || key === 'videoBitrate') {
+          value = formatBitrate(value);
+        }
```

Comment:
I think this should avoid the assignment to `value` even though it's not treated as a `const`.

```suggestion
        if (key === 'audioBitrate' || key === 'videoBitrate') {
          return (
            <DescriptionListItem
              key={key}
              title={title}
              data={formatBitrate(value)}
            />
          );
        }
```